garage/envs/base.py

```diff
@@ -25,10 +25,14 @@ class GarageEnv(gym.Wrapper, Parameterized, Serializable, metaclass=ABCMeta):
     Args: env (gym.Env): the env that will be wrapped
     """
 
-    def __init__(self, env):
-        Serializable.quick_init(self, locals())
+    def __init__(self, env=None, env_name=""):
```

Comment:
Good idea using a single wrapper. I wonder if the rest of the wrappers (e.g. Normalized, Noisy) could inherit from this.